packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -284,6 +285,39 @@ Messages.CacheRetrievalResult retrieveLostImage() {
     return result.build();
   }
 
+  public void chooseMediaFromGallery(
+      @NonNull Messages.ImageSelectionOptions imageOptions,
+      @NonNull Messages.VideoSelectionOptions videoOptions,
+      @NonNull Boolean usePhotoPicker,
+      @NonNull Messages.Result<List<String>> result) {
+    if (!setPendingOptionsAndResult(imageOptions, videoOptions, result)) {
+      finishWithAlreadyActiveError(result);
+      return;
+    }
+
+    launchPickMediaFromGalleryIntent(usePhotoPicker);
+  }
+
+  private void launchPickMediaFromGalleryIntent(Boolean useAndroidPhotoPicker) {
+    Intent pickMediaIntent;
+    if (useAndroidPhotoPicker && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+      pickMediaIntent =
+          new ActivityResultContracts.PickMultipleVisualMedia()
+              .createIntent(
+                  activity,
+                  new PickVisualMediaRequest.Builder()
+                      .setMediaType(ActivityResultContracts.PickVisualMedia.ImageAndVideo.INSTANCE)
+                      .build());
+    } else {
+      pickMediaIntent = new Intent(Intent.ACTION_GET_CONTENT);
+      pickMediaIntent.setType("*/*");
```

Comment:
Won't `ACTION_GET_CONTENT` with `*/*` allow literally any file type, not just media?